charts/tidb-lightning/secret.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-storage-secret
+type: Opaque
+stringData:
+  rclone.conf: |
+    [s3]
+    type = s3
+    provider = AWS
+    env_auth = false
+    access_key_id = <my-access-key>
+    secret_access_key = <my-secret-key>
+    region = us-east-1
+
+    [ceph]
+    type = s3
+    provider = Ceph
+    env_auth = false
+    access_key_id = <my-access-key
+    secret_access_key = <my-secret-key>
+    endpoint = <ceph-object-store-endpoint>
```

Comment:
add `region = :default-placement` for ceph

Reply:
I don't think this field is required.

Reply:
No, this field is required. Refer rook/rook#2945